aurora/aurora-war/src/main/webapp/look/jsp/TopBar.jsp

```diff
@@ -303,47 +307,54 @@ window.USERSESSION_PROMISE.then(function() {
         </c:if>
         <c:if test="${not isAnonymous}">
```

Comment:
The check here of isAccessGuest seems to be missing.
If it is the case, isAccessGuest checks performed into this `c:if` must disapear.